### PR TITLE
Readds Plasmusketry

### PR DESCRIPTION
**Pull request by HanSolo1519**
For the biannual Yuma chapter Brotherhood reenactor's clan meeting of 2277, an unnamed scribe derived an improved musket from a wild "Laser Musket". Nobody knows how they had plasma technology five hundred and four years ago, however, these newfangled, finicky devices, prone to overheating, seem to pack quite a punch.

## About The Pull Request
Plasma muskets were stricken from the code in #142 alongside a list of other guns for "serving 0 purpose", despite being one of the few craftable weapons with uniqueness _and_ utility that stands up in faction combat.

TLDR: Readds plasma musket If someone spends time making a 3 shot self-exploding plasma shitgun because someone nabbed the actual plasma rifle first, so be it. 

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: undoes the red text in #142 relating to plasmusketry,
/:cl:

### CODE REVIEW

**Comment by HanSolo1519** [comment]
After a couple drinks and listening to vega's schizorants at 5 in the morning (god bless that dude) I chose to revert the plasmusket ammo crafting to the state it's always been.

It's kinda uncalled for to make the ammo literally the most time-consuming ammo to obtain. It's literally a hobogun lol.

**Comment by companoo** [comment]
can this please be merged lol
